flake.nix

```diff
@@ -1,71 +1,72 @@
 {
   description = "sofyr";
-  inputs = {
-    nixpkgs.url = "github:nixos/nixpkgs/a3d847c3bd3a3b75b3057d7b3730d3308dd8fd59";
+  inputs = rec {
+    nixpkgs.url = "github:nixos/nixpkgs?ref=master";
     flake-utils.url = "github:numtide/flake-utils";
-    souffle-haskell.url = "github:luc-tielen/souffle-haskell";
+    souffle-haskell.url = "github:smunix/souffle-haskell?ref=fix.57";
```

Comment:
Once luc-tielen/souffle-haskell#59 and luc-tielen/souffle-haskell#57 has been merged, update the `url` above